src/content/lesson/asynchronous-algorithms-async-await.es.md

```diff
@@ -195,37 +204,40 @@ myPomise.then((obj) => {
 });
 console.log(myPomise);
 
-/*CONSOLE OUTPUT:
-	>objeto de promesa // devolverá un objeto de promesa
+/*RESULTADO CONSOLADO:
+	>Objeto de promesa // devolverá un objeto de promesa
 	>"Estaba resuelto"
 */
 ```
 ***
 ### Resolver y rechazar funciones
+
 + ***Resolver*** se utiliza para cambiar el estado de una promesa de pendiente a cumplida.
```

Comment:
we should keep the function name in english: `resolve` and `reject` because it is a reserved word.